### PR TITLE
Add update directive

### LINKED ISSUE #6: Status should be Obsoletes RFC 9127
The document status should be obsoletes RFC 9127

### PR DESCRIPTION
**Pull request by mjethanandani**

### CODE REVIEW

File: draft/draft-ietf-bfd-rfc9127-bis-00.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <?rfc tocdepth="6"?>
 <?rfc symrefs="yes"?>
 <?rfc sortrefs="yes"?>
-<rfc category="std" docName="draft-ietf-bfd-rfc9127bis-latest" ipr="trust200902">
+<rfc category="std" docName="draft-ietf-bfd-rfc9127-bis-00" ipr="trust200902" obsoletes="9127">
```
Comment (Reshad-Rahman) [inline draft/draft-ietf-bfd-rfc9127-bis-00.xml]: Should this be bis-01 (and the file too)? or maybe I misunderstand the generated files.

Comment (mjethanandani) [reply draft/draft-ietf-bfd-rfc9127-bis-00.xml]: You are right. It is partly automagically, and partly assigned. I fixed the assigned part, and now it should generate a bis-01 file. However, if you look at the diffs, you will see more differences than you care for.

Let me upload the rfcdiff file which would be easier to read/compare.

**Review by Reshad-Rahman — APPROVED** [review]
All good, thanks again.

**Review by jhaas-pfrc — APPROVED** [review]
The results of the merge looks fine.

**Pull request merged**